search/includes/classes/class-queue.php

```diff
@@ -381,6 +385,12 @@ public function queue_objects( $object_ids, $indexable_slug = 'post', $options =
 			return;
 		}
 
+		$indexable = Indexables::factory()->get( $indexable_slug );
+
+		if ( ! $indexable || ! \ElasticPress\Elasticsearch::factory()->index_exists( $indexable->get_index_name() ) ) {
```

Comment:
```suggestion
		if ( ! $indexable || ! $indexable->index_exists() ) {
```

For simplification since it calls `\ElasticPress\Elasticsearch::factory()->index_exists( $indexable->get_index_name() )` already